root/url/utility/isGreyedOut.js

```diff
@@ -7,10 +7,17 @@
  * later version: http://www.gnu.org/licenses/gpl-2.0.txt
  */
 
-const greyRegExp = new RegExp('^https?://(?:www\.)?decoda\.com/');
+/*
+ * NB: No URLs currently need to be greyed out;
+ * this should always return false until that changes
+ */
+
+// const greyRegExp = new RegExp('');
```

Comment:
I think I'd prefer just changing this to something like `/(?!)/`, which matches nothing, to avoid having to revert all this stuff whenever we need or don't need this.